reporter-web-app/src/components/SummaryView.js

```diff
@@ -32,12 +32,12 @@ import LicenseStatsTable from './LicenseStatsTable';
 import RuleViolationsTable from './RuleViolationsTable';
 import {
     getOrtResult,
-    getSummaryDeclaredLicenses,
-    getSummaryDeclaredLicensesChart,
-    getSummaryDeclaredLicensesFilter,
-    getSummaryDetectedLicenses,
-    getSummaryDetectedLicensesChart,
-    getSummaryDetectedLicensesFilter,
+    getSummaryDeclaredLicensesProcessed,
```

Comment:
I wonder if it'd be better to just use the term `declaredLicenses` (without processed) for the declared licenses all over the place. I assume the only interesting thing about the `raw` declared licenses is the set of `unmapped` ones. So how about replacing `declaredLicensesProcessed` + `declaredLicenses` with the following?
1. declaredLicenses
2. unmappedDeclaredLicenses

Reply:
I think using `declaredLicensesProcessed` is fine as that's exactly the field name we use in the Kotlin model.

Reply:
@sschuberth what do you think then about this?

1. declaredLicensesProcessed
2. unmappedDeclaredLicenses

Reply:
What I'm having a problem with is this: AFAIU the web-app reporter currently nowhere uses the `unmapped` part of `declaredLicensesProcessed`. Maybe *that* is already the problem, but that's also another topic. Currently only the (mapped) part of `declaredLicensesProcessed` is used, and `declaredLicenses`. And if using those two, I'd prefer to stick to the field names as they are in Kotlin code.

If the web-app reporter would be changed to use only `ProcessedDeclaredLicense.spdxExpression` and `ProcessedDeclaredLicense.unmapped`, then the names could be e.g. `declaredLicensesProcessed` and `declaredLicensesProcessedUnmapped`.

Reply:
thanks, makes sense!

Reply:
Actually I  do use `unmapped` part of `declaredLicensesProcessed` it's called `webAppPackage.declaredLicensesUnmapped`  and `spdxExpression` is called `webAppPackage.declaredLicensesSpdxExpression`.

- `unmapped` in the WebApp UI: https://github.com/oss-review-toolkit/ort/blob/master/reporter-web-app/src/components/PackageLicenses.js#L76
- `spdxExpression` in the WebApp UI: https://github.com/oss-review-toolkit/ort/blob/master/reporter-web-app/src/components/PackageLicenses.js#L65
`unmapped` and `spdxExpression`in the WebApp model:  https://github.com/oss-review-toolkit/ort/blob/master/reporter-web-app/src/models/WebAppPackage.js#L141-L168.

Reply:
And what's their relation to `declaredLicensesProcessed` used here?

Reply:
when I implemented this I was thinking of calling `declaredLicensesProcessed` `declaredLicensesMapped` but then I realized that it's not only mapping but also curated so that's how I ended up on `Processed`.